src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,27 @@
+package lotto.model
+
+import lotto.model.LottoNumber.Companion
+
+class Lotto(private val numbers: List<Int>) {
```

Comment:
java record 와 비슷한 코틀린 만의 data class 를 활용해볼 수 있겠습니다!

Reply:
```suggestion
class Lotto(private val numbers: Set<Int>) {
```
로또 숫자들의 중복을 방지할 수 있도록 Set 자료구조를 사용해보면 어떨까요?
그럼 검증문에서 distinct() 함수를 사용하지 않아도 되겠습니다